bundles/turn-combat/lib/CombatController.js

```diff
@@ -127,19 +128,25 @@ module.exports = class CombatController {
 
       p.character.combatData = {};
 
-      if (p.character.isNpc) {
-        this.gameState.MobManager.removeMob(p.character);
-        continue;
-      }
-
-      // at the end of combat put downed players at 1 HP
       if (condition === 'downed') {
-        p.character.setAttribute('health', 1);
-      }
-
-      p.character.emit('combatEnd');
-      Broadcast.sayAt(p.character, '-> The battle is over!');
-      Broadcast.prompt(p.character);
+        if (p.character.isNpc) {
+            this.gameState.MobManager.removeMob(p.character);
+            continue;
+        }
+        else
+        //put downed players at 1 HP
+        {
```

Comment:
JS Formatting: I would put the brace immediately after `else`, like `else {`, then the comment.

Reply:
Modified to match the recommended format(5c344ea)